traceapp/vis.go

```diff
@@ -14,6 +14,8 @@ import (
 	_ "sourcegraph.com/sourcegraph/appdash/sqltrace"
 )
 
+// ErrTimelineItemValidation is returned by timelineItem.Valid when either
+// timelineItem.Label or timelime.ItemFullLabel are empty.
 var ErrTimelineItemValidation = errors.New("timeline item validation error")
```

Comment:
Totally agree as well, addressed on https://github.com/chris-ramon/appdash/commit/a1846dd95511253cac8038357b69b2f50daba985

Thanks for reviewing this guys :+1: